FFXIVClientStructs/FFXIV/Client/System/File/FileAccessPath.cs

```diff
@@ -0,0 +1,16 @@
+namespace FFXIVClientStructs.FFXIV.Client.System.File;
+
+[StructLayout(LayoutKind.Explicit, Size = 0x210)]
+public unsafe struct FileAccessPath {
+    [FieldOffset(0x000)] public fixed char Buffer[260];
+    [FieldOffset(0x208)] public char* LongStringPtr;
+
+    public string AsString {
+        get {
+            if (LongStringPtr == null)
+                fixed (char* p = Buffer)
+                    return new string(p);
+            else return new string(LongStringPtr);
+        }
+    }
```

Comment:
I'd suggest overriding C#'s `ToString()` instead.
```suggestion
    public override string ToString() {
        if (LongStringPtr == null)
            fixed (char* p = Buffer)
                return new string(p);
        else
            return new string(LongStringPtr);
    }
```